Server.MirForms/Account/PlayerInfoForm.cs

```diff
@@ -133,5 +133,122 @@ private void OpenAccountButton_Click(object sender, EventArgs e)
             form.ShowDialog();
         }
 
+        private void FlagSearchBox_TextChanged(object sender, EventArgs e)
+        {
+            TextBox flagTextBox = (TextBox)sender;
+
+            if (string.IsNullOrWhiteSpace(flagTextBox.Text))
+            {
+                ResultLabel.Text = string.Empty;
+                return;
+            }
+
+            if (int.TryParse(flagTextBox.Text, out int flagIndex))
+            {
+                if (flagIndex >= 0 && flagIndex < Character.Flags.Length)
+                {
+                    bool flagValue = Character.Flags[flagIndex];
+
+                    if (flagValue)
+                    {
+                        ResultLabel.Text = $"Flag {flagIndex} ACTIVE.";
```

Comment:
change "ACTIVE." to "is active"
change "INACTIVE." to "is inactive"

so it matches the output format of quests

Reply:
This has been adjusted now